jupyterhub_config.py

```diff
@@ -306,7 +306,7 @@
 #c.JupyterHub.load_groups = {}
 
 ## Specify path to a logo image to override the Jupyter logo in the banner.
-#c.JupyterHub.logo_file = ''
+c.JupyterHub.logo_file = '../cylc-ui/dist/img/logo.png'
```

Comment:
Should we ship a copy of the logo in cylc-uiserver instead of loading it from cylc-ui? I think that path may fail if the user has cylc-ui somewhere else, or if the virtual env directory doesn't match?

(just finishing another code, will give it a try in a bit)

Reply:
Yes we probably should.  I just did it this way as a quick test, as we already refer to `../cylc-ui/dist` in one other place in `jupyterhub_config.py`.